Source/cmake/OptionsGTK.cmake

```diff
@@ -86,6 +83,8 @@ else ()
     WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_RESOURCE_USAGE PRIVATE OFF)
 endif ()
 
+WEBKIT_OPTION_DEFINE(USE_SKIA "Whether to use Skia instead of Cairo." PRIVATE OFF)
```

Comment:
Lets move this to a multi-port option and place it in `WebKitCommon.cmake`. Remove the same define from `OptionsWPE.cmake`.